cmd/crc/cmd/podman_env.go

```diff
@@ -25,32 +19,6 @@ var podmanEnvCmd = &cobra.Command{
 func runPodmanEnv(args []string) error {
 	// See issue #961; Currently does not work on Windows in combination with the CRC vm.
 	return errors.New("Currently not supported.")
-
-	userShell, err := shell.GetShell(forceShell)
-	if err != nil {
-		return fmt.Errorf("Error running the podman-env command: %s", err.Error())
-	}
-
-	ipConfig := machine.IpConfig{
-		Debug: isDebugLog(),
-	}
-
-	if err := exitIfMachineMissing(); err != nil {
-		return err
-	}
-
-	result, err := machine.New().Ip(ipConfig)
-	if err != nil {
-		return err
-	}
-
-	output.Outln(shell.GetPathEnvString(userShell, constants.CrcBinDir))
-	output.Outln(shell.GetEnvString(userShell, "PODMAN_USER", constants.DefaultSSHUser))
-	output.Outln(shell.GetEnvString(userShell, "PODMAN_HOST", result.IP))
-	output.Outln(shell.GetEnvString(userShell, "PODMAN_IDENTITY_FILE", constants.GetPrivateKeyPath()))
-	output.Outln(shell.GetEnvString(userShell, "PODMAN_IGNORE_HOSTS", "1"))
-	output.Outln(shell.GenerateUsageHint(userShell, "crc podman-env"))
-	return nil
```

Comment:
@guillaumerose This is not going to dead in near future, we had this functionality and then because of issue with podman we just put `not supported`.

Reply:
Added back